nipype/interfaces/fsl/epi.py

```diff
@@ -853,8 +855,15 @@ def __init__(self, **inputs):
                        "instead"), DeprecationWarning)
         return super(EddyCorrect, self).__init__(**inputs)
 
+    def _list_outputs(self):
```

Comment:
if the `name_source` and `name_template` works above, this function won't be necessary. do run `make specs` to regenerate the auto tests for this interface.